CHANGELOG.md

```diff
@@ -14,6 +14,14 @@ The format is based on [Keep a Changelog][keepachangelog] and this project adher
 ### Changed
```

Comment:
Where is `Added` section with description about all added classes?

Reply:
Sorry, I forgot. Added 29caa24043c1ef560e153aa938561ff08d7fd793